Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -2237,7 +2237,7 @@ RenderStyle HTMLInputElement::createInnerTextStyle(const RenderStyle& style)
     adjustInnerTextStyle(style, textBlockStyle);
 
     textBlockStyle.setWhiteSpaceCollapse(WhiteSpaceCollapse::Preserve);
-    textBlockStyle.setTextWrap(TextWrap::NoWrap);
+    textBlockStyle.setTextWrapMode(TextWrapMode::NoWrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
No, we only want to set the longhands of `white-space` which are `white-space-collapse` and `text-wrap-mode`. `text-wrap-style` should keep its default value of `auto` (there's no situation where the UA really needs to balance text by default usually).